src/main/java/it/gov/pagopa/payments/service/PartnerService.java

```diff
@@ -984,26 +996,39 @@ private PaymentOptionModelResponse getReceiptPaymentOption(
       String noticeNumber,
       String idPa,
       String creditorReferenceId,
+      boolean isStandIn,
       PaymentOptionModel body,
       ReceiptEntity receiptEntity)
       throws FeignException, URISyntaxException, InvalidKeyException, StorageException {
     PaymentOptionModelResponse paymentOption = new PaymentOptionModelResponse();
     try {
-      paymentOption = gpdClient.receiptPaymentOption(idPa, noticeNumber, body);
-      // creates the PAID receipt
-      if (PaymentOptionStatus.PO_PAID.equals(paymentOption.getStatus())) {
+      paymentOption = gpdClient.sendPaymentOptionReceipt(idPa, noticeNumber, body);
+
+      boolean isNotACA = true; // default SERVICE_TYPE_GPD
+      if(paymentOption.getServiceType() != null)
+        isNotACA = !paymentOption.getServiceType().equals(SERVICE_TYPE_ACA);
```

Comment:
use curly brackets 🤓

Reply:
I don't like curly brackets when the if statement has only 1 line, but to be uniform I'll adapt it.